packages/office-ui-fabric-react/src/components/pickers/BasePicker.types.ts

```diff
@@ -20,7 +21,7 @@ export interface IBasePicker<T> {
 
 // Type T is the type of the item that is displayed
 // and searched for by the people picker. For example, if the picker is
-// displaying persona's than type T could either be of Persona or Ipersona props
+// displaying persona's than type T could either be of Persona or IPersona props
```

Comment:
Small typo: shouldn't I be `...displaying persona's then type...` instead of `...displaying persona's than type...`?

Reply:
Fixed it. Thanks for taking a pass. All I did is fixed the squiggly lines I have from a grammar extension I have in VSCode, but it only checks for word typos and not sentence semantic constructs. 👍